app/renderer/js/pages/preference/new-server-form.js

```diff
@@ -58,7 +59,7 @@ class NewServerForm extends BaseComponent {
 			});
 		}, errorMessage => {
 			this.$saveServerButton.children[0].innerHTML = 'Connect';
-			alert(errorMessage);
+			dialog.showErrorBox(errorMessage.title, errorMessage.content);
```

Comment:
Have you tested this? Error returned from Line 145 doesn't have title or content keys in it.

Reply:
Ah okay, I must've missed this while testing. 
Thanks, I'll check all error messages and make sure the title and content keys are present in each error message. 

Will edit this commit directly.

Reply:
Okay, so pretty much the only clean way I know of to do this is to create new `Error` objects and then assign the properties `title` and `content` to them, which I feel is overkill for a feature that wasn't required in the first place (I just didn't like using `alert`). 

If you agree, then I can just drop this commit and go back to using `alert` with simple error strings. 

cc @akashnimare

Reply:
(Or we could create message objects with `title` and `content` props, `stringify` them when throwing the error, and then parse them on the receiving end, but I think this is really messy).

Reply:
Alert sounds okay to me.